extensions/wikia/Sitemap/SpecialSitemap_body.php

```diff
@@ -69,7 +69,20 @@ public function __construct( $name = 'Sitemap' ) {
 	 * @param $subpage Mixed: subpage of SpecialPage
 	 */
 	public function execute( $subpage ) {
-		global $wgMemc, $wgRequest, $wgOut, $wgEnableSitemapXmlExt;
+		global $wgMemc, $wgRequest, $wgOut, $wgEnableSitemapXmlExt, $wgSitemapXmlExposeInRobots;
+
+		// Force new SitemapXML on old URL, because Google expects it under the old address
+		// and otherwise we'd need to update hundreds of items in Google Search Console
+		if ( !empty( $wgEnableSitemapXmlExt )
+			&& !empty( $wgSitemapXmlExposeInRobots )
+			&& strpos( $subpage, '-oldsitemapxml-' ) === false
+		) {
+			$wgOut->disable();
+			$response = F::app()->sendRequest( 'SitemapXml', 'index', [ 'path' => $subpage ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "index" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)